src/components/hello_world.vue

```diff
@@ -1,61 +1,79 @@
 <template>
```

Comment:
Go ahead and move the demo stuff into it's own demo component in the demos folder, then add that demo component to the version of hello_world from develop